extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -486,13 +518,11 @@ const OHIFCornerstoneViewport = React.memo(props => {
 
   return (
     <React.Fragment>
-      <div className="viewport-wrapper">
+      <div className="viewport-wrapper" data-rendered={dataRendered}>
         <ReactResizeDetector
-          handleWidth
-          handleHeight
-          skipOnMount={true} // Todo: make these configurable
-          refreshMode={'debounce'}
-          refreshRate={200} // transition amount in side panel
+          skipOnMount={true}
+          refreshMode="throttle"
```

Comment:
see my other message that has the video displaying the flickering effect

Reply:
I see the issue, debounce is definitely wrong in the current setup, but maybe there is another parameter I can use.  debounce on trailing edge would work